samples/DotNet/Azure.Messaging.ServiceBus/BasicSendReceiveTutorialwithFilters/BasicSendReceiveTutorialWithFilters/SendReceive.cs

```diff
@@ -0,0 +1,138 @@
+﻿//using Microsoft.Azure.ServiceBus;
+//using Microsoft.Azure.ServiceBus.Core;
+using System;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Azure.Messaging.ServiceBus;
+using System.Linq;
+
+namespace BasicSendReceiveTutorialWithFilters
+{
+    class SendReceive
+    {
+        public string ServiceBusConnectionString;
+        public string TopicName;
+        public string[] Subscriptions;
+        public string[] Store;
+        public int NrOfMessagesPerStore;
+
+        public async Task SendMessages()
+        {
+            try
+            {
+
+                await using (ServiceBusClient client = new ServiceBusClient(ServiceBusConnectionString))
+                {
```

Comment:
Is this scope needed? Could be done with `await using var client = new ServiceBusClient(...);`